# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/hyprland/hotfix-man.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- hyprland-source/Makefile.vanilla	2023-03-06 20:02:00.963843920 +0100
+++ hyprland-source/Makefile	2023-03-06 20:02:36.657844837 +0100
@@ -179,7 +179,7 @@
 	cp ./assets/wall_4K.png ${PREFIX}/share/hyprland
 	cp ./assets/wall_8K.png ${PREFIX}/share/hyprland
 
-	install -Dm644 -t ${PREFIX}/share/man/man1 ./docs/*.1
+	cp  ./docs/*.1 ${PREFIX}/share/man/man1
 
 cleaninstall:
 	echo -en "make cleaninstall has been DEPRECATED, you should avoid using it in the future.\nRunning make install instead...\n"
